.env

```
@@ -13,9 +13,11 @@ DATABASE_URL="postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_URI}/$
 
 # Github App env
 GITHUB_CLIENT_SECRET=""
+# An SSH key (private) or its path that can push to the index repo
+SSH_KEY=""
```

nit: could we rename?
```suggestion
GITHUB_SSH_KEY=""
```